pages/common/ps.md

```diff
@@ -13,3 +13,11 @@
 - Search for a process that matches a string:
 
 `ps aux | grep {{string}}`
+
+- List all processes of the current user in extra full format:
```

Comment:
The extra full format is already covered by `auxww`. Let's just keep the next example.

Reply:
It's not really the same

using `auxww`

    USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
    suboptim    5503  0.0  0.1  22056  5888 pts/11   Ss   12:28   0:00 bash

using `-F`

    UID        PID  PPID  C    SZ   RSS PSR STIME TTY          TIME CMD
    suboptim  5503  3129  0  5514  5888   0 12:28 pts/11   00:00:00 bash

Reply:
I see. It is indeed not a perfect match. What I meant was that it's more of an extended format like `auxww`. I wanted to have just one option with an extended format. But if you feel this is a helpful option from a beginner's perspective, we can keep it.

Reply:
As the usage of `ps` is really divers I would suggest to add one more advanced example for beginner's, which shows how to find the parent process `PPID` easily.
I can't really judge if it is helpfull for most of the beginners or not. Why not leaving it in till someone propose an example maybe more useful for beginner's?

Reply:
Since we are now just at 5 examples, it should not be a big issue.